README.md

```diff
@@ -28,7 +28,7 @@ If you know about more interesting 3D data sources, please open issues with link
 -  Montréal (lod2 + textures)
    http://donnees.ville.montreal.qc.ca/dataset/maquette-numerique-batiments-citygml-lod2-avec-textures
 
-## Czech Republic
+## Czech
```

Comment:
"Czech" is an adjective - did you mean Czechia? :-)

Reply:
Yea, that. Just remove the snobby "Republic" part!